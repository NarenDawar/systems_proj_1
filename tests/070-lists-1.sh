# The operators "&&" and "||" shall have equal precedence and shall be evaluated 
# with left associativity. For example, both of the following commands write solely bar to standard output:
false && echo skipped || echo test andor ok
true || echo skipped && echo test orand ok
