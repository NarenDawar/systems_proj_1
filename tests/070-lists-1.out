test andor ok
test orand ok
